hazelcast/src/main/java/com/hazelcast/map/impl/AbstractMapServiceContextSupport.java

```diff
@@ -152,8 +152,14 @@ public void interceptAfterRemove(String mapName, Object value) {
     }
 
     @Override
-    public String addInterceptor(String mapName, MapInterceptor interceptor) {
-        return mapServiceContext.getMapContainer(mapName).addInterceptor(interceptor);
+    public void addInterceptor(String id, String mapName, MapInterceptor interceptor) {
+        mapServiceContext.getMapContainer(mapName).addInterceptor(id, interceptor);
```

Comment:
Yeah, I know what it called, the purpose of this change was not refactoring but a change of wrong behavior. Will fix it anyway